name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java

```diff
@@ -256,6 +258,23 @@ public String getText(Object element)
         }));
         portfolioColumns.addColumn(column);
 
+        column = new Column("ref_cash_bal", "Balance of reference account", SWT.RIGHT, 100); //$NON-NLS-1$
+        column.setLabelProvider(new ColumnLabelProvider()
+        {
+            CurrencyConverter converter = new CurrencyConverterImpl(factory, getClient().getBaseCurrency());
```

Comment:
The converter is not used. The balance will be shown in the account currency. Is this intended?

Reply:
> The converter is not used. The balance will be shown in the account currency. Is this intended?

Yeah, it's clearly was a copy-paste issue, where label provider was initially copied from security balance column, but the getText() then was copied from AccountListView, leaving converter unused.

Thanks for taking care of the patch!